main.cpp

```diff
@@ -137,7 +139,10 @@ int main(int argn, char** argv) {
       return EXIT_FAILURE;
     }
 
-    if (print_entry_point || print_shader_stage || print_source_file) {
+    if (ci_mode) {
+      // CI Mode just is used to supress logs when running in CI
```

Comment:
nit: can you clarify this?

Reply:
```suggestion
      // When running CI we want to just test that SPIRV-Reflect doesn't crash,
      // The output is not important (as there is nothing to compare it too)
      // This hidden flag is here to allow a way to suppress the logging in CI to only the shader name
      // (otherwise the logs noise and GBs large)
```

How about this?